modules/installation-configuration-parameters.adoc

```diff
@@ -3235,7 +3235,7 @@ The name of one or more failures domains.
 
 |platform:
   nutanix:
-    defaulatMachinePlatform:
+    defaultMachinePlatform:
```

Comment:
@yanhua121 @tuxtof  @sgaoshang You helped review the [initial PR to add this code](https://github.com/openshift/openshift-docs/pull/69618) to the docs. Are you able to take a look at this change? It seems reasonable to me.